mantidproject/mantid
# A new algorithm to integrate peaks given by an EPP table

This PR introduces a new algorithm `IntegrateEPP` which integrates a workspace around the elastic peak positions given by an EPP table. The algorithm is planned to be used in ILL's detector diagnostics as well as a backend in [`ComputeCalibrationCoefVan`](http://docs.mantidproject.org/nightly/algorithms/ComputeCalibrationCoefVan-v1.html).

**To test:**

1. Have a workspace with a nice elastic line. For example, `<mantid-build>/ExternalData/Testing/Data/UnitTest/ILL/IN4/084446.nxs`
2. Apply [`FindEPP`](http://docs.mantidproject.org/nightly/algorithms/FindEPP-v2.html) to the workspace.
3. Run `IntegrateEPP`.

Check that the integrals are correct and the integration intervals (the X bin edges in the output workspace) sound.

Fixes #19609.

*Release notes were updated accordingly.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
